3ds2/src/main/res/layout/view_da_authentication.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    tools:parentTag="android.widget.LinearLayout">
+
+    <ImageView
+        android:id="@+id/ivLogo"
+        android:layout_width="wrap_content"
+        android:layout_height="40dp"
+        android:layout_gravity="center"
+        android:layout_marginTop="44dp"
+        android:src="@drawable/ic_adyen_fingerprint"
+        tools:ignore="ContentDescription" />
+
+    <TextView
+        android:id="@+id/tvTitle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:layout_marginHorizontal="24dp"
+        android:layout_marginTop="24dp"
+        android:fontFamily="sans-serif-medium"
+        android:gravity="center"
+        android:textSize="20sp"
+        tools:text="@string/checkout_3ds2_da_authentication_title" />
+
+    <TextView
+        android:id="@+id/tvDescription"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:layout_marginHorizontal="24dp"
+        android:layout_marginTop="8dp"
+        android:gravity="center"
+        android:textSize="14sp"
+        tools:text="@string/checkout_3ds2_da_authentication_description" />
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Why is this wrapper needed? We're doing the same in `QrCodeView`, so it should be working here as well

Reply:
The idea of this wrapper is to define percentage value from the width instead of setting some dp value. As for `QrCodeView` we use the second approach - let's use it here as well, I'll remove the wrapper